services/QuillLMS/app/models/blog_post.rb

```diff
@@ -146,6 +146,7 @@ def related_posts
     BlogPost
       .where(topic: topic)
       .where.not(id: id)
+      .where(draft: false)
```

Comment:
There's already a scope on line 91 that does this. Let's use the scope, or delete the scope

Reply:
This would also be an classic place to use a default scope, but i'm personally not a fan of default scopes because they can make debugging difficult if you forget that they exist.